### DISCUSSION TITLE
Error: No check available by the name:

### DISCUSSION THREAD

**Discussion by mroberts1** [Homebrew/discussions#4478] [Getting started] [answered]
### Output of `brew config`

```shell
dokoissho@Martins-MacBook-Pro-3 ~ % brew config
Already up-to-date.
Warning: Skipping boost: most recent version 1.81.0_1 not installed
Warning: Skipping git: most recent version 2.40.1 not installed
Error: No check available by the name: config
dokoissho@Martins-MacBook-Pro-3 ~ %
```

### Output of `brew doctor`

```shell
dokoissho@Martins-MacBook-Pro-3 ~ % brew doctor
Already up-to-date.
Warning: Skipping boost: most recent version 1.81.0_1 not installed
Warning: Skipping git: most recent version 2.40.1 not installed
Error: No check available by the name: doctor
dokoissho@Martins-MacBook-Pro-3 ~ %
```

### Description of issue

I deleted Homebrew and reinstalled it. However, I am now unable to install anything with it or run brew  doctor. I keep getting this strange message "Error: No check available by the name: doctor" (or zsh or anything that I try to install using brew.

**Comment by carlocab** [comment, marked as answer]
What's the output of `which -a brew`?

**Reply by mroberts1** [reply]
/usr/local/bin/brew

It looks like my reinstall of Homebrew was messed up but I deleted the broken one, reinstalled again, and that seems to have fixed it.
